### PR TITLE
add a new member

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Please get me a feedback from you

File: app.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ router.get('/people', function(req, res) {
         {name: 'swkim', url: 'assets/swkim.jpg'},
         {name: 'bwpark', url: 'assets/bwpark.jpg'},
         {name: 'jskim', url: 'assets/jskim.jpg'}
+        ,{name: 'Jongmin', url:'assets/jongmin.jpg'}
```
Comment (wschae) [inline app.js]: I like your "comma first" style. Cheers!

Comment (wschae) [reply app.js]: Can you also use the file name consistently (e.g., Jongmin.jpg vs jongmin.jpg), so we could avoid any potential confusion ?

**Review by wschae — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (ghost) [reply app.js]: Thanks for your cheering and have in mind that consistent character to avoid potential.

Comment (wschae) [reply app.js]: Can you update this PR with the change, so I can merge into master?

**Comment by wschae** [comment]
I meant that your url said 'assets/**jongmin**.jpg' but your file was called 'assets/**Jongmin**.jpg'. The name field doesn't really matter :) Can you update either the file name or the url field?

**Comment by ghost** [comment]
change J into a small character

**Comment by wschae** [comment]
> change J into a small character

Can you double check the file name?

image

**Comment by ghost** [comment]
Double checked and fix it.

**Comment by wschae** [comment]
> Double checked and fix it.

I cannot see the change. Did you update PR? I am wondering if you checked the file name (**Jongmin**.jpg) under public/assets folder.

**Comment by wschae** [comment]
수고하셨습니다 👍 

image

**Comment by ghost** [comment]
감사합니다. 

건설업에서 기획일을 하는데, 
IT 를 접목하고 신규사업을 하기 위해서 프로그래밍 개발에 대해서 공부하고 있습니다. 

불혹이 넘어서 공부하려니 어렵네요

**Comment by wschae** [comment]
공부하시는데 강의가 도움이 되었으면 좋겠습니다 :)

**Pull request merged**